src/ranged.cpp

```diff
@@ -32,6 +32,7 @@ const skill_id skill_throw( "throw" );
 const skill_id skill_gun( "gun" );
 const skill_id skill_melee( "melee" );
 
+static inline projectile make_gun_projectile( const item &gun );
```

Comment:
Just FYI, inline is almost always pointless except in a few places where it's required by scoping shennanigans.  It's a hint to the compiler that is almost always ignored, and for good reason, programmers don't have good insight into which code *should* be inlined.